sources/packages/backend/apps/queue-consumers/src/services/cas-supplier/cas-supplier.service.ts

```diff
@@ -1,26 +1,38 @@
 import { Injectable } from "@nestjs/common";
 import { InjectRepository } from "@nestjs/typeorm";
-import { SystemUsersService } from "@sims/services";
-import { CASSupplier, SupplierAddress, SupplierStatus } from "@sims/sims-db";
+import { CASSupplier, SupplierStatus } from "@sims/sims-db";
 import { ProcessSummary } from "@sims/utilities/logger";
-import { Repository, UpdateResult } from "typeorm";
-import { CASService } from "@sims/integrations/cas/cas.service";
-import { CustomNamedError } from "@sims/utilities";
+import { Repository } from "typeorm";
 import {
+  CASService,
   CASAuthDetails,
-  CASSupplierResponse,
-  CASSupplierResponseItem,
-  CASSupplierResponseItemAddress,
-} from "@sims/integrations/cas/models/cas-supplier-response.model";
+  formatAddress,
+  formatPostalCode,
+} from "@sims/integrations/cas";
+import { CustomNamedError, isAddressFromCanada } from "@sims/utilities";
 import { CAS_AUTH_ERROR } from "@sims/integrations/constants";
+import {
+  CASEvaluationResult,
+  CASEvaluationStatus,
+  NotFoundReason,
+  PreValidationsFailedReason,
+} from "./cas-supplier.models";
+import {
+  CASActiveSupplierNotFoundProcessor,
+  CASPreValidationsProcessor,
+  CASActiveSupplierFoundProcessor,
+  CASEvaluationResultProcessor,
+} from "./cas-evaluation-result-processor";
 
 @Injectable()
 export class CASSupplierIntegrationService {
   constructor(
     private readonly casService: CASService,
-    private readonly systemUsersService: SystemUsersService,
     @InjectRepository(CASSupplier)
     private readonly casSupplierRepo: Repository<CASSupplier>,
```

Comment:
May not be a part of this PR but a question here: Can we not extend this class from `RecordDataModelService<T>` and pass the `repo` to the superclass and initialize it that way?

Reply:
`RecordDataModelService` is no longer recommended and should not be used.